BangazonWorkforce/bangazon.sql

```diff
@@ -454,7 +454,23 @@ VALUES
 INSERT INTO EmployeeTraining
```

Comment:
This insert statement gives me the following error:

```
Msg 110, Level 15, State 1, Line 454
There are fewer columns in the INSERT statement than values specified in the VALUES clause. The number of values in the VALUES clause must match the number of columns specified in the INSERT statement.
```

Reply:
using original SQL